pkg/wolfssl/include/user_settings.h

```diff
@@ -73,6 +73,7 @@ int strncasecmp(const char *s1, const char * s2, unsigned int sz);
 #define TFM_TIMING_RESISTANT
 #define USE_CERT_BUFFERS_2048
 #define NO_RC4
+#define WOLFSSL_STATIC_PSK
```

Comment:
Sorry for not stating this clearly in my last comment. I actually meant something like below, so that you can choose to compile wolfSSL with or without pre-shared key support at compile time with `USEMODULE += wolfcrypt_psk` just like ECC.

```C
#undef WOLFSSL_STATIC_PSK
#ifdef MODULE_WOLFCRYPT_PSK
  #define WOLFSSL_STATIC_PSK
#endif
```

Reply:
Fixed, added module `USEMODULE += wolfssl_psk` . Updated dtls makefile to show the possible options for DH/PK, and the PSK alternative.